spec/index.bs

```diff
@@ -10689,13 +10689,25 @@ It must only be included by interfaces which also include those mixins.
                 For each pair of ({{GPUIndex32}} |index|, {{GPUBindGroupLayout}} |bindGroupLayout|) in
                 |pipeline|.{{GPUPipelineBase/[[layout]]}}.{{GPUPipelineLayout/[[bindGroupLayouts]]}}:
                 - Let |bindGroup| be |encoder|.{{GPUBindingCommandsMixin/[[bind_groups]]}}[|index|].
+                - Let |dynamicOffsets| be |encoder|.{{GPUBindingCommandsMixin/[[dynamic_offsets]]}}[|index|].
                 - |bindGroup| must not be `null`.
                 - |bindGroup|.{{GPUBindGroup/[[layout]]}} must be [=group-equivalent=] with |bindGroupLayout|.
-                - For each |entry| of |bindGroup|.{{GPUBindGroup/[[entries]]}}:
-                    - If |entry|.{{GPUBindGroupEntry/[[prevalidatedSize]]}} is `false`:
-                        - [$effective buffer binding size$](|entry|.{{GPUBindGroupEntry/resource}}) must be &ge; [=minimum buffer binding size=]
-                            of the binding variable in |pipeline|'s shader that corresponds to |entry|.
-            - [$Encoder bind groups alias a writable resource$](|encoder|, |pipeline|) must be `false`.
```

Comment:
This check seems to have been inadvertently removed?

Reply:
Good catch!